wiki/Tournaments/OWC/2020/zh.md

```diff
@@ -115,38 +115,65 @@ osu! 2020 世界杯由 osu! team 和多位社区成员举办。
 | ![][flag_VN] | **越南** | **[Tuon](https://osu.ppy.sh/users/6673790)**, [Ui chan](https://osu.ppy.sh/users/5449433), [\_Kuroni\_](https://osu.ppy.sh/users/6159305), [Hoaq](https://osu.ppy.sh/users/7696512), [Huukun](https://osu.ppy.sh/users/8114571), [FukutoTojido](https://osu.ppy.sh/users/8266808), [sindes19](https://osu.ppy.sh/users/11021073) |
 
-## 比赛时间表：半决赛
+## 比赛时间表：决赛
 
-### 2020 年 11 月 21 日，星期六
+### 2020 年 11 月 28 日，星期六
 
 | A 队 | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |  |
 | --: | :-- | :-: | :-: | :-: | :-: |
-| 香港 ![][flag_HK] | ![][flag_AU] 澳大利亚 | **18:00 UTC+08** | 18:00 UTC+08 | 21:00 UTC+11 | ² |
-| 中国 ![][flag_CN] | ![][flag_ID] 印度尼西亚 | **20:00 UTC+08** | 20:00 UTC+08 | 19:00 UTC+07 | ² |
-| 韩国 ![][flag_KR] | ![][flag_RU] 俄罗斯 | **21:00 UTC+08** | 22:00 UTC+09 | 16:00 UTC+03 | ² |
-| 巴西 ![][flag_BR] | ![][flag_TW] 台湾 | **22:30 UTC+08** | 11:30 UTC-03 | 22:30 UTC+08 | ² |
+| 德国 ![][flag_DE] | ![][flag_AU] 澳大利亚 | **19:00 UTC+08** | 12:00 UTC+01 | 22:00 UTC+11 | ² |
+| 日本 ![][flag_JP] | ![][flag_BR] 巴西 | **22:00 UTC+08** | 23:00 UTC+09 | 11:00 UTC-03 | ² |
 
-### 2020 年 11 月 22 日，星期日
+### 2020 年 11 月 29 日，星期日
 
 | A 队 | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |  |
 | --: | :-- | :-: | :-: | :-: | :-: |
-| 加拿大 ![][flag_CA] | ![][flag_JP] 日本 | **01:00 UTC+08** | 12:00 UTC-05 | 02:00 (Sun.) UTC+09 | ¹ |
-| 美国 ![][flag_US] | ![][flag_DE] 德国 | **03:00 UTC+08** | 12:00 UTC-07 | 20:00 UTC+01 | ¹ |
-| 中国 ![][flag_CN] | ![][flag_AU] 澳大利亚 | **15:00 UTC+08** | 15:00 UTC+08 | 18:00 UTC+11 | ³ |
-| 印度尼西亚 ![][flag_ID] | ![][flag_AU] 澳大利亚 | **17:00 UTC+08** | 16:00 UTC+07 | 20:00 UTC+11 | ³ |
-| 中国 ![][flag_CN] | ![][flag_HK] 香港 | **20:00 UTC+08** | 20:00 UTC+08 | 20:00 UTC+08 | ³ |
-| 印度尼西亚 ![][flag_ID] | ![][flag_HK] 香港 | **20:00 UTC+08** | 19:00 UTC+07 | 20:00 UTC+08 | ³ |
-| 韩国 ![][flag_KR] | ![][flag_BR] 巴西 | **21:30 UTC+08** | 22:30 UTC+09 | 10:30 UTC-03 | ³ |
-| 韩国 ![][flag_KR] | ![][flag_TW] 台湾 | **21:30 UTC+08** | 22:30 UTC+09 | 21:30 UTC+08 | ³ |
-| 俄罗斯 ![][flag_RU] | ![][flag_TW] 台湾 | **21:30 UTC+08** | 16:30 UTC+03 | 21:30 UTC+08 | ³ |
-| 俄罗斯 ![][flag_RU] | ![][flag_BR] 巴西 | **23:00 UTC+08** | 18:00 UTC+03 | 12:00 UTC-03 | ³ |
+| 美国 ![][flag_US] | ![][flag_CA] 加拿大 | **07:00 UTC+08** | 16:00 UTC-07 | 18:00 UTC-05 | ¹ |
+| 巴西 ![][flag_BR] | ![][flag_AU] 澳大利亚 | **09:00 UTC+08** | 22:00 (星期六) UTC-03 | 12:00 UTC+11 | ³ |
+| 日本 ![][flag_JP] | ![][flag_AU] 澳大利亚 | **18:00 UTC+08** | 19:00 UTC+09 | 21:00 UTC+11 | ³ |
+| 日本 ![][flag_JP] | ![][flag_DE] 德国 | **21:00 UTC+08** | 22:00 UTC+09 | 14:00 UTC+01 | ³ |
+
+### 2020 年 11 月 30 日，星期一
+
+| A 队 | B 队 | 比赛时间 | A 队当地时间 | B 队当地时间 |  |
+| --: | :-- | :-: | :-: | :-: | :-: |
+| 巴西 ![][flag_BR] | ![][flag_DE] 德国 | **01:00 UTC+08** | 14:00 UTC-03 | 18:00 UTC+01 | ³ |
```

Comment:
Would note that both brazil and germany here are playing on a sunday local time.